src/webview/js/rewriteImageUrls.js

```diff
@@ -32,11 +32,10 @@ const rewriteImageUrls = (auth: Auth, element: Element | Document = document) =>
   imageTags.forEach(img => {
     // Get the raw `src` value from the DOM. (We can't easily use `img.src`,
     // since it's absolutized by the browser.)
-    /* $FlowFixMe (upstream 'getNamedItem' is mistyped) */
-    const actualSrc: string | null = img.attributes.getNamedItem('src')?.value;
+    const actualSrc = img.getAttribute('src');
```

Comment:
(We discussed this in person.)

I see this as an example of a variable/expression having a boring reasonable type, a type that's what you would hope it would be and what makes the code after it make sense; and I like the practice to be that that's the *unmarked* case, with nothing specifically calling the reader's attention to it.

The specific type can also be confirmed when it matters by hovering in the IDE over the variable (either here or where used), or the `getAttribute` method. But basically I see leaving it out as a promise that it's boring and you don't need to do that to understand the code, and I think that promise is fulfilled here.

You're wary in particular of the DOM API being one that's full of surprises, and don't like trusting that the type is reasonable. You don't love leaving the type implicit here, but you're OK with it.